rag/prompt_templates.py

```diff
@@ -0,0 +1,82 @@
+from langchain.schema import SystemMessage
+from langchain.prompts import (
+    HumanMessagePromptTemplate,
+    ChatPromptTemplate,
+    MessagesPlaceholder,
+)
+
+optional_preprompt = """
```

Comment:
this prompt is probably superseded by the below prompt? Or is the idea to keep it as it's a much more slimmed down version of the below. If so then we should update the extent of the knowledge base (which we will be updating to satisfy Renbees use)

Reply:
This is the old "system" prompt. In the earlier implementation of the chatbot, the prompt was the pre-prompt concatenated with the query, there was no system prompt used.

Reply:
In effect, I've kept the old prompt for reuse